experimenter/experimenter/nimbus_ui_new/templates/nimbus_experiments/table.html

```diff
@@ -110,11 +111,14 @@ <h1 class="modal-title fs-5" id="createModalLabel">
           </td>
           <td>
             {% if experiment.start_date %}
-              {{ experiment.start_date|date:"Y-m-d" }} -
-              <br>
-              {{ experiment.computed_end_date|date:"Y-m-d" }}
-              <br>
-              ({{ experiment.computed_duration_days }} days)
+              {{ experiment.start_date|date:"Y-m-d" }}
+            {% else %}
+              N/A
+            {% endif %}
+          </td>
+          <td>
+            {% if experiment.computed_end_date %}
+              {{ experiment.computed_end_date|date:"Y-m-d" }}({{ experiment.computed_duration_days }} days)
```

Comment:
Just noticed this typo:
```suggestion
              {{ experiment.computed_end_date|date:"Y-m-d" }} ({{ experiment.computed_duration_days }} days)
```

Reply:
Hrmmm I can't seem to apply this change, probably because it's on @RJAK11 's fork.  Let's just land this and we can do that as a followup when @RJAK11 is back 🙏

Reply:
It's my fork so I could probably do it, but that's fine too -- it's just a space!